public/javascripts/export1Table.js

```diff
@@ -0,0 +1,119 @@
+var pickTranslation = function(songID, translationID, checked) {
+    if (checked) {
+        console.log(translationID)
+        m.request({
+                method: 'POST',
+                url: '/user/playlist/' + playlistName + '/export1',
+                data: {
+                    playlistID: playlistID,
+                    songID: songID,
+                    translationID: translationID
+                }
+            })
+            .then(function(res) {
+                console.log(res.msg)
+            })
+    } else {
+        m.request({
+                method: 'DELETE',
+                url: '/user/playlist/' + playlistName + '/export1',
+                data: {
+                    playlistID: playlistID,
+                    songID: songID,
+                    translationID: translationID
+                }
+            })
+            .then(function(res) {
+                console.log(res.msg)
+            })
+    }
+
+}
+
+
+var langLabelArr = function(translationss) {
+    var lang = []
+    translationss.forEach((tr) => {
```

Comment:
Change this to not use the translationss convention. Use a different variable name, maybe even translations2d or something that is more descriptive. The purpose is to reduce typos and make the code easier to read and understand. Also since you're using the => fat arrow, we may need to use babeljs to make this client side code work. Alternatively you'd change it to use the old function () notation.